src/Evolution/Executables/ScalarWave/EvolveScalarWave.hpp

```diff
@@ -57,6 +57,7 @@
 #include "Parallel/RegisterDerivedClassesWithCharm.hpp"
```

Comment:
In the future can you please make sure you only format the lines you're changing? The commit message makes this sound like a very short but there is a lot going on in this commit. Try to make the life of the reviewer as easy as possible :)

Reply:
@wthrowe , @nilsdeppe , we are ready for a final review.